### PR TITLE
unification of ecosystem overview and terminology

### PR DESCRIPTION
**Pull request by brentzundel**
This PR unifies the language used in the ecosystem overview (section 1.2) and terminology definitions (section 2) to partially addres issue #481

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/508.html" title="Last updated on Apr 2, 2019, 2:25 AM UTC (ede9787)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/508/1ad7698...brentzundel:ede9787.html" title="Last updated on Apr 2, 2019, 2:25 AM UTC (ede9787)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -352,21 +351,20 @@ <h3>Ecosystem Overview</h3>
           </dd>
           <dt><a>verifier</a></dt>
           <dd>
-A role an <a>entity</a> might perform by requesting and receiving a
-<a>verifiable presentation</a> that proves the <a>holder</a> possesses the
-required <a>verifiable credentials</a> with certain characteristics. Example
-verifiers include employers, security personnel, and websites.
+A role an <a>entity</a> might perform by receiving one or more
+<a>verifiable presentations</a> for processing. Example verifiers include
```
Comment (David-Chadwick) [inline index.html]: verifiable presentations are optional not mandatory. So reword to

A role an <a>entity</a> might perform by receiving one or more <a>verifiable credentials</a>,
optionally inside a <a>verifiable presentation</a>, for processing.

Comment (brentzundel) [reply index.html]: Changes made in [52d48bc](https://github.com/w3c/vc-data-model/pull/508/commits/52d48bc92cf0b371d91a42c7ac8a3394bfba51b4)

File: index.html [hunk 2]
```diff
@@ -352,21 +351,22 @@ <h3>Ecosystem Overview</h3>
           </dd>
           <dt><a>verifier</a></dt>
           <dd>
-A role an <a>entity</a> might perform by requesting and receiving a
-<a>verifiable presentation</a> that proves the <a>holder</a> possesses the
-required <a>verifiable credentials</a> with certain characteristics. Example
-verifiers include employers, security personnel, and websites.
+A role an <a>entity</a> might perform by receiving one or more
+<a>verifiable credentials</a>, optionally inside a
+<a>verifiable presentations</a>, for processing. Example verifiers include
```
Comment (ken-ebert) [inline index.html]: "inside a verifiable presentations" could better be "inside a verifiable presentation" or "inside verifiable presentations"

Comment (brentzundel) [reply index.html]: fixed in [ede9787](https://github.com/w3c/vc-data-model/pull/508/commits/ede978791a48a2f05d8998c7f19ef204e20aed98)

**Pull request merged**